src/main/java/com/meilisearch/sdk/http/URLBuilder.java

```diff
@@ -1,5 +1,8 @@
 package com.meilisearch.sdk.http;
 
+import lombok.Getter;
+
+@Getter
 public class URLBuilder {
     private StringBuilder routes, params;
 
```

Comment:
What do you think about creating a constructor which receives the root route?

Reply:
Yes! I was thinking of doing that too, but it's a bigger modification and it would involve a lot of changes, so I didn't do it. Did I do it in another PR, or maybe I could open an issue about it WDYT?

Reply:
I suggested it in my new review, but I'll also approve the PR, feel free to implement the changes or let it to another issue :)